docs/getting_started.md

```diff
@@ -93,16 +93,13 @@ make install
 go run ./cmd/agent/main.go
 ```
 
-In the near future we will provide individual Docker images for each component,
-as well as the Kubernetes templates.
-
 ## Migrating from Zipkin
 
-Collector service exposes Zipkin compatible REST API `/api/v1/spans` and can be enabled by
-`--collector.zipkin.http-port=9411`. It supports Thrift and JSON format.
-Agent uses `TBinaryProtocol` and is available on `UDP` port `5775`.
+Collector service exposes Zipkin compatible REST API `/api/v1/spans` and `/api/v2/spans`.
```

Comment:
We should clarify : both thrift and JSON? For both versions?

Reply:
done